src/ploneintranet/workspace/workspacecontainer.py

```diff
@@ -15,3 +15,12 @@ class WorkspaceContainer(Container):
     A folder to contain WorkspaceFolders
     """
     grok.implements(IWorkspaceContainer)
+
+try:
```

Comment:
Sounds good!

Since attachments are not the focus of this pull request I would propose to merge now and refactor the attachment code afterwards.